unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -332,14 +375,20 @@ export default class ActionBinder {
     return true;
   }
 
-  async handleSingleFileUpload(file, eventName) {
-    const fileData = { type: file.type, size: file.size, count: 1 };
+  async handleSingleFileUpload(file, eventName) {  
+    const sanitizedFileName = await this.sanitizeFileName(file.name); 
+
+    const newFile = new File([file], sanitizedFileName, { type: file.type, lastModified: file.lastModified });
+    const fileData = { type: newFile.type, size: newFile.size, count: 1 };
+
     this.dispatchAnalyticsEvent(eventName, fileData);
-    if (!await this.validateFiles([file])) return;
+
+    if (!await this.validateFiles([newFile])) return;
```

Comment:
Not sure if this is just on my side, but please remove the additional new line spacing in between here as well. Thanks!

Reply:
@arugupta1992 could we please get this updated as well?